docker-compose.yml

```diff
@@ -27,6 +28,7 @@ services:
       PGSQL_USER: "${PGSQL_USER}"
       PGSQL_PASSWORD: "${PGSQL_PASSWORD}"
       PHP_VERSION: "${PHP_VERSION}"
+      SSMTP_REWRITEDOMAIN: "${SSMTP_REWRITEDOMAIN}"
```

Comment:
suggest SSMTP_REWRITE_DOMAIN

Reply:
Same as above, I named it to match the configuration parameter:

```
# Where will the mail seem to come from?
#RewriteDomain=
```